uploadToLoki.js

```diff
@@ -2,13 +2,16 @@
 /* eslint-disable no-console, no-loop-func, max-len */
 require('dotenv').config();
 
-
-const axios = require('axios').default;
-const fs = require('fs');
+const axios = require("axios").default;
+const fs = require("fs");
+const grayMatter = require("gray-matter");
```

Comment:
This library parses YAML blocks so that the SQL can contain what's needed for the query object